libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -548,4 +568,14 @@ class BreezSDKMapper {
         case .expired: return "expired"
         }
     }
+
+    static func valueOf(reverseSwapStatus: ReverseSwapStatus) -> String {
+        switch(reverseSwapStatus) {
+        case .initial: return "initial"
+        case .inprogress: return "inprogress"
+        case .cancelled: return "cancelled"
+        case .completedseen: return "completedseen"
+        case .completedconfirmed: return "completedconfirmed"
```

Comment:
These should match the equivalent kotlin enum values which are the lowercase of this enum:
```
enum class ReverseSwapStatus {
    INITIAL,IN_PROGRESS,CANCELLED,COMPLETED_SEEN,COMPLETED_CONFIRMED;
}
```

Reply:
Done.